rpcs3/Emu/Io/LogitechG27.cpp

```diff
@@ -1516,4 +1545,11 @@ void usb_device_logitech_g27::interrupt_transfer(u32 buf_size, u8* buf, u32 endp
 	}
 }
 
+bool usb_device_logitech_g27::set_configuration(u8 cfg_num)
+{
+	// setting it through libusb_set_configuration on linux fails, but when passthrough does it through control transfer, it doesn't fail
+	logitech_g27_log.todo("emulator sets config num %u outside of control transfer, fail it for now", cfg_num);
```

Comment:
usb_device_logitech_g27::set_configuration isn't called outside control transfer, it is called by 
- usb_device_logitech_g27::control_transfer : https://github.com/RPCS3/rpcs3/blob/3894c903bc6ee7b1fe267a45f8727b725f2eabb7/rpcs3/Emu/Io/LogitechG27.cpp#L128
-> usb_device_emulated::control_transfer : https://github.com/RPCS3/rpcs3/blob/3894c903bc6ee7b1fe267a45f8727b725f2eabb7/rpcs3/Emu/Io/usb_device.cpp#L286
-> usb_device::set_configuration